helmfile.yaml

```diff
@@ -15,6 +15,8 @@ releases:
 
 - name: graphprotocol-node-index
   chart: ./charts/graphprotocol-node
+  labels:
+    part: node-index
```

Comment:
IMHO it is better to call it "component"

Reply:
Renamed